### DISCUSSION TITLE
"Classic" score display mode applies osu!standard scoring to everything

### DISCUSSION THREAD

**Discussion by ianpep** [ppy/osu#18406] [Ideas]
### Type

Game behaviour

### Bug description

As described in the title, setting/changing the "Score Display Mode" under the gameplay tab to "classic" in settings changes scoring across all gamemodes, and also across playlists. 
![image](https://user-images.githubusercontent.com/54043451/170150963-3a63e4e6-b975-436f-b9fa-f34e360c73c3.png)
(pictured, mania scores wildly above what they should be)
![image](https://user-images.githubusercontent.com/54043451/170151361-ca701645-25d6-4597-afae-0ac407b39452.png)
(pictured, scores of a song selected within a playlist)
![image](https://user-images.githubusercontent.com/54043451/170151461-8482ccef-1482-406c-aad8-47b2ad7ac994.png)
As seen above, this does not affect the overall leaderboard of a playlist.
![image](https://user-images.githubusercontent.com/54043451/170151790-f262fb27-f1ae-4461-af08-58c8f7d5162e.png)
(pictured, another mania score with auto played without any modifiers. i have noticed inconsistently that it adds score to plays with certain modifiers on mania, although i cannot recreate it at this time)

### Version

2022.523.0-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/8766980/updater.log)
[database.log](https://github.com/ppy/osu/files/8766981/database.log)
[network.log](https://github.com/ppy/osu/files/8766982/network.log)
[performance.log](https://github.com/ppy/osu/files/8766983/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8766984/runtime.log)

**Comment by smoogipoo** [comment]
This is intentional for now, but we may reconsider in the future. The setting name should probably not remain as "classic" if it remains this way.

**Reply by ianpep** [reply]
I'm trying to point out that it is applying in some cases where it probably shouldn't, such as in the beatmap spotlights, where the overall scoring and the per beatmap scoring differs between leaderboards.

**Reply by smoogipoo** [reply]
Ah, in that case this would be a duplicate of https://github.com/ppy/osu/issues/17824?
